site/public/js/details.js

```diff
@@ -129,3 +131,37 @@ function inquiry_update() {
         button.textContent = 'Grade Inquiry Only: Off';
     }
 }
+
+function checkProperTicks() {
+    fillAllCheckboxes(true);
+    const selectedColumns = JSON.parse(decodeURIComponent(Cookies.get('grading_details_columns')));
+    for (let i = 0; i < selectedColumns.length; i++) {
+        document.getElementById(`toggle-${selectedColumns[i]}`).checked = false;
+    }
```

Comment:
Why not use a [`.forEach(...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) here instead of iterating through indices?